tests/accessibility/pa11y.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs')
+const path = require('path')
+const archiver = require('archiver')
+const pa11y = require('pa11y')
+const htmlReporter = require('pa11y-reporter-html')
+
+const reportsPath = path.join(__dirname, 'reports.zip')
+
+let archive
+
+const tests = [
+  // pa11y('http://localhost:3000/form-example/3') <- add URL to test here
```

Comment:
can you uncomment this and use the ID 1 as this is in the db in source. I also believe this currently fails the check so would be good to get these fixed now

Reply:
Sure - I purposely kept a broken URL here to show report generation, but I suppose there's no point committing broken code!